pkg/services/live/live.go

```diff
@@ -868,6 +868,94 @@ func (g *GrafanaLive) HandleChannelRulesListHTTP(c *models.ReqContext) response.
 	})
 }
 
+type configInfo struct {
```

Comment:
@FZambia -- I don't think this is the final way we should do this, but having a single place with types, description and a sample object will be super helpful